bspump/http/web/server.py

```diff
@@ -470,7 +471,15 @@ def render_html_output(self, json_data):
     def format_json_to_html(self, json_data):
         fields_html = []
         for key, value in json_data.items():
-            if isinstance(value, dict):
+            if key == "button":
```

Comment:
This is going to conflict with a case in which an API just wants to return some key named `button`. We want to be able to display arbitrary JSON with this so this could be problematic. What you can do is create a serializable class named `Button` that serializes as json to the string `"button"` and then check if the key is of type `Button`